### PR TITLE
Add Toggle for NewModule Form and skipCRDs Support in Backend

### LINKED ISSUE #602: Add `skipCRDs` option to Module definition
Each Module is deployed from a template that can be a Helm chart. Helm charts can contain a list of CRDs that need to be installed in order for the application to work or provide its capability.

However, managing CRDs through Helm is opinionated, so the scope of the issue is to disable creating CRDs defined in a template when a Module is created.

**Describe the solution you'd like**

The [Module definition](https://github.com/cyclops-ui/cyclops/blob/main/cyclops-ctrl/api/v1alpha1/module_types.go) should store whether CRD creation should be skipped and taken into account in the [reconciliation loop](https://github.com/cyclops-ui/cyclops/blob/5760ffeff3a4deb3a818aa7a20f9141150e11878/cyclops-ctrl/internal/modulecontroller/module_controller.go#L199). Also, to make it easier for users, the skip CRDs option should be available  in the advanced section when creating a new Module and passed to the cyclops-ctrl on Module submit.

### PR DESCRIPTION
**Pull request by kom-senapati**
closes #602 

## 📑 Description
- Added support for the skipCRDs option in the backend reconciliation logic.
- Added a toggle switch for the skipCRDs option in the frontend NewModule form.

## ✅ Checks
- [X] I have tested my code (provide screenshots or screen recordings of a working solution)
- [X] I have performed a self-review of my code

## Screenshots

image

### CODE REVIEW

**Comment by kom-senapati** [comment]
@quest-bot loot #602

File: cyclops-ui/src/components/pages/NewModule/NewModule.tsx [hunk 1]
```diff
@@ -666,6 +669,21 @@ const NewModule = () => {
                       ))}
                     </Select>
                   </Form.Item>
+                  <Form.Item
+                    name="skipCRDs"
```
Comment (petar-cvit) [inline cyclops-ui/src/components/pages/NewModule/NewModule.tsx:673]: Rname to `cyclops_module_skip_crds`

**Review by petar-cvit — CHANGES_REQUESTED** [review]
The skip crds values from the UI is not properly passed when creating a module and is never applied.

Also, since you are changing the CRD, run `make manifests` to generate manifests in `cyclops-ctrl/config/crd/bases`, and make sure the value is properly mapped in the kubernetes cluster

File: cyclops-ui/src/components/pages/NewModule/NewModule.tsx (hunk 1 above)
Comment (kom-senapati) [reply cyclops-ui/src/components/pages/NewModule/NewModule.tsx]: Renamed

**Comment by kom-senapati** [comment]
@petar-cvit Can you tell me the changes required in the backend

**Comment by kom-senapati** [comment]
```sh
$ make manifests
make: *** No rule to make target 'manifests'.  Stop.
```

It's giving this output when running inside the `cyclops-ctrl/config/crd/bases` directory.

**Pull request closed without merging**